tests/smoke.sh

```diff
@@ -90,10 +96,14 @@ function rpc_node_try {
     try_silent "curl -X POST -H \"Content-Type: application/json\" --data '{\"jsonrpc\":\"2.0\",\"method\":\"eth_syncing\",\"params\":[],\"id\":1}' 127.0.0.1:4$1" "Could not connect to RPC of node $1."
 }
 
-# Errors if an RPC connection to the graph is not possible. Works only with chain IDs, not names.
-function rpc_graph_try {
-    info "Checking RPC connection to graph $1."
-    try_silent "curl -X GET 127.0.0.1:5$1" "Could not connect to RPC of graph $1."
+function rpc_origin_sub_graph_try {
+    info "Checking RPC connection to origin sub graph for $2 chain on node for $1."
+    try_silent "./node_modules/.bin/ts-node tests/Graph/SubGraphDeployment/origin-verifier.ts $1 6$2" "Origin sub graph for $2 was expected to be deployed on $1, but wasn't."
```

Comment:
instead of concatenating like 6$2 should it pass exact port. To make it more clean.